src/components/SystemMenu/SystemMenu.jsx

```diff
@@ -105,13 +117,16 @@ function SystemMenu({ showTutorial }) {
             <button type="button" onClick={() => onClick(openFeedback)}>
               Send Feedback
             </button>
+
```

Comment:
Why new lines?

Reply:
Found it a bit hard to see where the separated sections started and ended as the `<HorizontalDelimiter />` is on the same indention level as all the buttons. This was a whim to make that a bit easier

But I see below that I wasnät even consistent. I'm ok with removing them again